service/storage_proxy.hh

```diff
@@ -159,7 +159,6 @@ class storage_proxy : public seastar::async_sharded_service<storage_proxy>, publ
 public:
     using clock_type = lowres_clock;
     struct config {
-        db::hints::host_filter hinted_handoff_enabled = {};
```

Comment:
Instead of removing this you should have made it `utils::updateable_value_source`.